src/jest.js

```diff
@@ -153,11 +153,13 @@ function _promiseOnlyChangedTestPaths(testRunner, config) {
   return Promise.all(testPathDirsAreGit)
     .then(function(results) {
       if (!results.every(function(result) { return result; })) {
+        /* eslint-disable no-throw-literal */
```

Comment:
Maybe we should throw `new Error` instead? I decided to disable the eslint-rule to not change any logic.